lib/wovnrb.rb

```diff
@@ -66,22 +68,26 @@ def call(env)
 
     def switch_lang(headers, body)
       translated_body = []
+
+      # Must use `.each` for to support multiple-chunks in Sinatra
       string_body = ''
       body.each { |chunk| string_body += chunk }
       html_body = Helpers::NokogumboHelper::parse_html(string_body)
```

Comment:
If we keep using `html_body` at this higher level, we should leverage it everywhere in `HtmlConverter` instead of using sometimes regular expressions and some other times Nokogumbo.